posts/2023-06-22-mfdataset-preprocess.ipynb

```diff
@@ -0,0 +1,8180 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5a572220-fae2-4250-ae4f-3b7621aad20f",
+   "metadata": {},
+   "source": [
+    "# More efficent use of `xarray.open_mfdataset()`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32b10f7c-8b63-4d33-b4c7-aefca37cd6c8",
+   "metadata": {},
+   "source": [
+    "A common workflow associated with data analysis using `xarray` is a \"chain\"-type workflow, where one operation leads to another, then to another, and so on until the final result is produced. Each operation produces a `DataSet` or `DataArray` that is a step closer to the final result.\n",
+    "\n",
+    "Most workflows of this type have a 'correct' order of operations that will minimise the amount of computation done on the dataset. Running the workflow in the 'wrong' order won't necessarily fail, it may just take longer. Whilst this isn't an issue with smaller models, when analysing the data produced large models like AUS2200, running these operations out-of-order can often result in hangs or crashes.\n",
```

Comment:
I would talk about 'datasets' rather than models here at least in "an issue with smaller models" as this applies to any datasets with lots of files